shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -116,12 +135,28 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val bestPcpPaxEstimate: Int = (ApiPax, ActPax, TranPax, MaxPax) match {
-    case (Some(apiPax), _, _, _) if !FeedSources.contains(LiveFeedSource) => apiPax
-    case (_, Some(actPax), Some(tranPax), _) if (actPax - tranPax) >= 0 => actPax - tranPax
-    case (_, Some(actPax), None, _) => actPax
-    case (Some(apiPax), _, _, _) => apiPax
-    case _ => 0
+  val bestPcpPaxEstimate: TotalPaxSource = (ApiPax, ActPax, TranPax, MaxPax, TotalPax) match {
+    case (_, _, _, _, totalPax) if totalPax.exists(tp => tp.feedSource == ScenarioSimulationSource && tp.pax > 0) =>
+      totalPax.find(tp => tp.feedSource == ScenarioSimulationSource)
+        .getOrElse(TotalPaxSource(0, UnknownFeedSource, None))
+    case (_, _, _, _, totalPax) if totalPax.exists(tp => tp.feedSource == LiveFeedSource && tp.pax > 0) =>
+      totalPax.find(tp => tp.feedSource == LiveFeedSource && tp.pax > 0)
+        .getOrElse(TotalPaxSource(0, UnknownFeedSource, None))
+    case (_, _, _, _, totalPax) if totalPax.exists(tp => tp.feedSource == ApiFeedSource && tp.splitSource.contains(ApiSplitsWithHistoricalEGateAndFTPercentages) && tp.pax > 0) =>
```

Comment:
Missing case for `ForecastFeedSource`, which is the next one after `LiveFeedSource`